.vscode/settings.json

```diff
@@ -6,5 +6,6 @@
   "files.trimTrailingWhitespace": true,
   "typescript.preferences.importModuleSpecifier": "project-relative",
   "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "files.eol": "\n"
+  "files.eol": "\n",
+  "typescript.tsdk": "node_modules\\typescript\\lib"
```

Comment:
```suggestion
  "typescript.tsdk": "node_modules/typescript/lib"
```
Does this work on non Windows?

Reply:
![image](https://github.com/user-attachments/assets/49f271d6-c989-471f-89f6-41d6374b4d6e)
these should be js files, should work for any platform

Reply:
The file path uses forward slashes which might not work on non Windows platforms

Reply:
fair, i've changed it to backslashes.